proxy/src/http/server.rs

```diff
@@ -0,0 +1,27 @@
+use anyhow::anyhow;
```

Comment:
Looks a bit out of scope according to commit message. Will this check be used by k8s healthcheck system? If so should we make changes to helm charts?

Reply:
My bad, it is existing code that was just moved from previous location and I didnt spot that